content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+
+Um Ingress pode ser configurado para fornecer URLs acessíveis externamente aos serviços, balanceamento de carga de tráfego, terminação SSL/TLS e oferecer hospedagem virtual baseada em nome.
+Um [controlador Ingress](/docs/concepts/services-networking/ingress-controllers) é responsável por atender o Ingress, geralmente com um balanceador de carga, embora também possa configurar seu roteador de borda ou frontends adicionais para ajudar a lidar com o tráfego.
+
+Um Ingress não expõe portas ou protocolos arbitrários. 
+Normalmente se usa um serviço do tipo [Service.Type=NodePort](/docs/concepts/services-networking/service/#type-nodeport) ou [Service.Type=LoadBalancer](/docs/concepts/services-networking/service/#loadbalancer) para expor serviços à Internet que não sejam HTTP e HTTPS.
+
+## Pré-requisitos
+
+Você deve ter um [controlador Ingress](/docs/concepts/services-networking/ingress-controllers) para satisfazer um Ingress. 
+Apenas a criação de um recurso Ingress não tem efeito.
+
+Você pode precisar instalar um controlador Ingress, como [ingress-nginx](https://kubernetes.github.io/ingress-nginx/deploy/). 
+Você pode escolher entre vários [controladores Ingress](/docs/concepts/services-networking/ingress-controllers).
+
+Idealmente, todos os controladores Ingress devem se encaixar na especificação de referência. 
+Na realidade, os vários controladores Ingress operam de forma ligeiramente diferente.
+
+{{< note >}}
+Certifique-se de revisar a documentação do seu controlador Ingress para entender as ressalvas de escolhê-lo.
+{{< /note >}}
+
+## O recurso Ingress
+
+Um exemplo mínimo do recurso Ingress:
+
+{{< codenew file="service/networking/minimal-ingress.yaml" >}}
+
+Um Ingress precisa dos campos `apiVersion`, `kind`, `metadata` and `spec`. 
+O nome de um objeto Ingress deve ser um nome de [subdomínio DNS válido](/pt-br/docs/concepts/overview/working-with-objects/names#dns-subdomain-names). 
+Para obter informações gerais sobre como trabalhar com arquivos de configuração, consulte como [instalar aplicações](/docs/tasks/run-application/run-stateless-application-deployment/), como [configurar contêineres](/docs/tasks/configure-pod-container/configure-pod-configmap/) e como [gerenciar recursos](/docs/concepts/cluster-administration/manage-deployment/). 
+O Ingress frequentemente usa anotações para configurar opções dependendo do controlador Ingress. Um exemplo deste uso é a [anotação rewrite-target](https://github.com/kubernetes/ingress-nginx/blob/master/docs/examples/rewrite/README.md). 
```

Comment:
I hate this annotation! :P Can we use another one? haha! Just kidding, if this is the example, let's stick with this one.